configs/org.kde.astroarch.desktop/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
what this file will do?

Reply:
This metadata.json file is used now instead of metdata.desktop.

The explanation can be found here :

https://develop.kde.org/docs/plasma/widget/properties/#metadatajson